config/server-config.yaml.example

```diff
@@ -46,7 +46,8 @@ database:
 
 identity:
   server:
-    issuer_url: http://keycloak:8080 # Use http://localhost:8081 instead for running minder outside of docker compose
+    issuer_url: http://localhost:8081
+    issuer_claim: http://localhost:8081/realms/stacklok
```

Comment:
We still want to have the default be the all-in-docker setup?

Reply:
I can set this whichever way -- we override the config value with a flag in docker-compose for the all-in-docker setup.